config/webpack.config.js

```diff
@@ -80,7 +83,7 @@ var configGenerator = (options) => {
     },
     plugins: [
       new webpack.DefinePlugin({
-          __BUILDTYPE__: options.buildtype
+          __BUILDTYPE__: JSON.stringify(options.buildtype)
```

Comment:
I'm not super knowledgeable about Travis, but I think that will just set NODE_ENV for the Node process that's running Webpack? Webpack itself also needs to be configured, through the define plugin, to replace NODE_ENV in the bundle it's building. I think it would look something like:

```
new webpack.DefinePlugin({
      __BUILDTYPE__: JSON.stringify(options.buildtype),
     'process.env': {
         'NODE_ENV': JSON.stringify(process.env.NODE_ENV)
     }
})
```